ledger/ledger-configuration/protobuf/com/daml/ledger/configuration/ledger_configuration.proto

```diff
@@ -32,7 +32,7 @@ message LedgerConfiguration {
   // ledger effective time and maximum record time of transactions.
   LedgerTimeModel time_model = 3;
 
-  // The maximum value for the ``deduplication_time`` parameter of command submissions
+  // The maximum value for the ``deduplication_duration`` parameter of command submissions
```

Comment:
This is a bit add odds with the new description in the `LedgerConfiguration`:
```
Command submissions with longer periods MAY get accepted though.
```
Can you adjust it to match what you wrote in `ledger_configuration_service.proto`?

Reply:
Addressed in #10851 -- let me know what you think.